WordPress/src/main/java/org/wordpress/android/ui/reader/discover/viewholders/ReaderPostViewHolder.kt

```diff
@@ -116,6 +115,7 @@ class ReaderPostViewHolder(
     }
 
     private fun updateDiscoverSection(state: ReaderPostUiState) {
+        uiHelpers.updateVisibility(image_discover_avatar, state.discoverSection != null)
```

Comment:
Shouldn't the condition be `state.discoverSection?.discoverAvatarUrl != null`, wdyt?

Reply:
Updated here 38ec932